appium-tests/android/android.spec.js

```diff
@@ -450,143 +306,110 @@ describe('Camera tests Android.', function () {
                             throw 'Couldn\'t find "Choose video" element.';
                         });
                 })
-                .then(win, fail)
+                .fail(saveScreenshotAndFail)
                 .deviceKeyEvent(4)
-                .sleep(2000)
-                .deviceKeyEvent(4)
-                .sleep(2000)
                 .elementById('action_bar_title')
                 .then(function () {
                     // success means we're still in native app
                     return driver
                         .deviceKeyEvent(4)
```

Comment:
It would be nice to have constants clarifying these `deviceKeyEvent` numbers. I'm guessing that this corresponds to pressing the back button?